aws-android-sdk-auth-google/src/main/AndroidManifest.xml

```diff
@@ -10,4 +10,4 @@
     <uses-permission android:name="android.permission.GET_ACCOUNTS" />
     <uses-permission android:name="android.permission.USE_CREDENTIALS" />
  
-<uses-sdk android:minSdkVersion="23" android:targetSdkVersion="25" /></manifest>
```

Comment:
This will also help resolve #471 since `minSdkVersion` in the `build.gradle` file for this package is set to 11